test/e2e/specs/onboarding/setup__videopress.ts

```diff
@@ -39,7 +39,21 @@ describe( DataHelper.createSuiteTitle( 'VideoPress Tailored Onboarding' ), () =>
 		} );
 
 		it( 'Click Get Started', async function () {
-			await Promise.all( [ page.waitForNavigation(), page.click( 'text=Get started' ) ] );
+			// Return a button "key" for the button that is found on page.
+			// Only 1 should be present.
+			const foundButtonKey = await Promise.any( [
+				page.waitForSelector( 'text=Get started' ).then( () => 'get-started-button' ),
+				page.waitForSelector( 'img[alt="Get a video portfolio"]' ).then( () => 'portfolio-button' ),
```

Comment:
@jgcaruso This is not quite recommended anymore by Playwright maintainers - it's suggested to use locators, which is able to significantly reduce flakiness. 

In addition, it is preferable to use accessible locators instead of CSS or text selectors.

Additionally, we do not prefer to use the `then` style of promises as dictated in our style guide.

Lastly, if matching on multiple possible selectors, Playwright has a built-in way to do that: await page.locator('button').click();.